micro-ui/web/micro-ui-internals/packages/modules/dss/CHANGELOG.md

```
@@ -0,0 +1,6 @@
+
+## [1.8.3]
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)